src/client/components/Dashboard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import DeckItem from './DeckItem';
+// import _ from 'underscore';
+import $ from 'jquery';
```

Comment:
Do we intend to use jQuery for anything but HTTP requests? If not then we may have an opportunity to refactor at some point using `fetch` with a polyfill for browsers other than Chrome and Mozilla.